src/Cake.Common/Tools/DotCover/DotCoverTool.cs

```diff
@@ -70,5 +70,22 @@ protected ProcessArgumentBuilder GetArguments(DotCoverSettings settings)
 
             return builder;
         }
+
+        /// <summary>
+        /// Get configuration full path from coverage settings
+        /// </summary>
+        /// <param name="settings">The settings</param>
+        /// <returns>The process arguments</returns>
+        protected ProcessArgumentBuilder GetConfigurationFileArgument(DotCoverSettings settings)
+        {
+            var builder = new ProcessArgumentBuilder();
```

Comment:
Any reason why this is added to a separate method and nit part of `GetArguments()`?

Reply:
Based on documentation for "cover" command
https://www.jetbrains.com/help/dotcover/dotCover__Console_Runner_Commands.html
"
It is possible to specify part of the parameters or all of them in the configuration file.
usage: dotCover cover <configuration file name> [parameters]
"
configuration file name should be right after command

Reply:
@mholo65 do you have some tips how can it be done in some other way which will better fit Cake in general?